toolkit/scripts/mariner-required-configs.json

```diff
@@ -0,0 +1,640 @@
+                "m",
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/2671"
+        },
+        "CONFIG_DELL_WMI_SYSMAN": {
+            "value": [
+                "m",
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/5406"
+        },
+        "CONFIG_TASK_DELAY_ACCT": {
+            "value": [
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": null
+        },
+        "CONFIG_DELL_SMBIOS": {
+            "value": [
+                "m",
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/2671"
+        },
+        "CONFIG_THERMAL_HWMON": {
+            "value": [
+                "m",
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/5417"
+        },
+        "CONFIG_TCG_TPM": {
+            "value": [
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": null
+        },
+        "CONFIG_HW_RANDOM_TPM": {
+            "value": [
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": null
+        },
+        "CONFIG_EDAC_SKX": {
+            "value": [
+                "m",
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/5487"
+        },
+        "CONFIG_NET_CLS_FLOWER": {
+            "value": [
+                "m",
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/5170"
+        },
+        "CONFIG_HIST_TRIGGERS": {
+            "value": [
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for auoms metrics",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/5292"
+        },
+        "CONFIG_INIT_ON_FREE_DEFAULT_ON": {
+            "value": [
+                "is not set",
+                ""
+            ],
+            "arch": [
```

Comment:
Not needed for this PR, but consider that this arch field could extend to more architectures in the future